core/src/main/scala/cats/syntax/validated.scala

```diff
@@ -33,6 +33,9 @@ final class ValidatedIdSyntax[A](private val a: A) extends AnyVal {
   def validNel[B]: ValidatedNel[B, A] = Validated.Valid(a)
   def invalid[B]: Validated[A, B] = Validated.Invalid(a)
   def invalidNel[B]: ValidatedNel[A, B] = Validated.invalidNel(a)
+
+  def validA[F[_], B]: Validated[F[B], A] = Validated.Valid(a)
```

Comment:
I'm wondering – should we keep `validA` in syntax once we agreed on removing it from `Validated`?

Reply:
agree, forgot to remove it 👍